src/dataflow-types/src/logging.rs

```diff
@@ -163,6 +164,27 @@ impl LogVariant {
                 .with_column("worker", ScalarType::Int64.nullable(false))
                 .with_column("time", ScalarType::Int64.nullable(false)),
 
+            LogVariant::Materialized(MaterializedLog::KafkaBrokerRTT) => RelationDesc::empty()
+                .with_column("consumer_name", ScalarType::String.nullable(false))
+                .with_column("source_id", ScalarType::String.nullable(false))
+                .with_column("dataflow_id", ScalarType::Int64.nullable(false))
+                .with_column("broker_name", ScalarType::String.nullable(false))
+                .with_column("min", ScalarType::Int64.nullable(false))
+                .with_column("max", ScalarType::Int64.nullable(false))
+                .with_column("avg", ScalarType::Int64.nullable(false))
+                .with_column("sum", ScalarType::Int64.nullable(false))
+                .with_column("cnt", ScalarType::Int64.nullable(false))
```

Comment:
could we name this `count`? does this match something else that we've already called `cnt`